File: Python/examples/PaymentAPI/cancelPayment.py
import os
import os.path
import sys
sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), os.path.pardir + os.path.sep + os.path.pardir)))
from dotenv import load_dotenv
import json

load_dotenv(os.path.pardir + os.sep + ".env")
eid = os.getenv("EID")
secret = os.getenv("SECRET")

from PaymentAPI import PaymentAPI

# Create a PaymentAPI object
api = PaymentAPI(eid, secret, debug=True)
payment = api.call(function="cancelPayment", data={"number":"4003208"})
print(json.dumps(payment, indent=4))

File: Deno/examples/PaymentAPI/cancelPayment.ts
import {QvicklyPaymentAPI, env} from "../../PaymentAPI.ts";

const paymentAPI = new QvicklyPaymentAPI(env["EID"], env["SECRET"]);
const payment = await paymentAPI.call("cancelPayment", {
  number: "12345",
});
console.log(payment);
